Wrappers.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "MassageChair.h"
+#include "CatShelter.h"
+#include "Hospital.h"
```

Comment:
Don't include what you don't use.
Some of these header files don't need to be included.
Use Forward Declaration instead
- [ ] revise